src/components/ScopedNotification/ScopedNotification.js

```diff
@@ -23,15 +34,7 @@ const ScopedNotification = (props) => {
       {...rest}
       center
       className={cx(sldsClasses)}
-      figureLeft={(
-        <Icon
-          icon="info"
-          sprite="utility"
-          size="small"
-          svgClassName={theme === 'dark' ? null : 'slds-icon-text-default'}
-          title="information"
-        />
-      )}
+      figureLeft={renderIcon({ theme, size: 'small' })}
```

Comment:
Still not an object in `defaultIconRenderer`. So it will break without passing this parameter